tests/api/route-services/projects.test.js

```diff
@@ -0,0 +1,79 @@
+const AWSMock = require('aws-sdk-mock');
+const AWS = require('../../../src/utils/requireAWS');
+
+const ProjectsService = require('../../../src/api/route-services/projects');
+const { NotFoundError, OK } = require('../../../src/utils/responses');
+
+describe('tests for the projects service', () => {
+  afterEach(() => {
+    AWSMock.restore('DynamoDB');
+  });
+
+  const mockDynamoUpdateItem = (jsData) => {
+    const dynamodbData = {
+      Attributes: AWS.DynamoDB.Converter.marshall({ projects: jsData }),
+    };
+    const getItemSpy = jest.fn((x) => x);
+    AWSMock.setSDKInstance(AWS);
+    AWSMock.mock('DynamoDB', 'updateItem', (params, callback) => {
+      getItemSpy(params);
+      callback(null, dynamodbData);
+    });
+    return getItemSpy;
+  };
```

Comment:
we have this piece of code everywhere where we need to mock DynamoDB -- it might be worth moving it to a single place and reusing it in this and other test files.